apps/state/test/state/routes_by_service_test.exs

```diff
@@ -80,6 +81,14 @@ defmodule State.RoutesByServiceTest do
       assert for_service_ids_and_types([@service.id], ["some_other_type"]) == []
       assert for_service_ids_and_types(["some_other_service"], [@route.type]) == []
     end
+
+    test "returns a list with no duplicates if multiple routes have the same type" do
```

Comment:
👍

Can we also add a test that there are no duplicates if there are multiple service IDs which have the same route? For example, if "service A" and "service B" both have "route 1".